### DISCUSSION TITLE
[Question]: deploy on Vercel

### DISCUSSION THREAD

**Discussion by hjfvhjmhj** [danny-avila/LibreChat#1057] [Q&A]
### What is your question?

If I want to deploy the project with Vercel, will it be successful?

### More Details

Hello, I would like to ask if this project can be deployed with vercel as a container, and if so, whether the environment variables are the same as render

### What is the main subject of your question?

Installation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
While not impossible, it will not be easy. This might put you on the right track tho:
https://www.geeksforgeeks.org/how-to-deploy-mern-application-on-vercel/

**Comment by johnfelipe** [comment]
i think is important have a vercel button deploy, pls add to roadmap
